### PR TITLE
chore(navsidebar): Added NavSidebar stories

### PR DESCRIPTION
**Pull request by greg-in-a-box**
Screen Shot 2022-11-28 at 6 05 56 PM
Screen Shot 2022-11-28 at 6 06 02 PM

### CODE REVIEW

**Review by megansmith-box — DISMISSED** [review]

File: src/components/nav-sidebar/NavSidebar.stories.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/* eslint-disable */
```
Comment (tjuanitas) [inline src/components/nav-sidebar/NavSidebar.stories.js:1]: what's this disabling?

**Review by tjuanitas — APPROVED** [review]

File: src/components/nav-sidebar/NavSidebar.stories.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+export const collapsible = () => (
+    <NavSidebar data-resin-component="leftnav">
+        <NavList
+            heading={
+                <NavListCollapseHeader onToggleCollapse={action('onToggleCollapse called')}>
+                    Collapse or Expand
+                </NavListCollapseHeader>
+            }
+            className="is-collapsible example-collapsible"
```
Comment (tjuanitas) [inline src/components/nav-sidebar/NavSidebar.stories.js:34]: where's the `example-collapsible` being used?

File: src/components/nav-sidebar/NavSidebar.stories.js (hunk 1 above)
Comment (greg-in-a-box) [reply src/components/nav-sidebar/NavSidebar.stories.js:1]: the linter was going off on the Link component not have a href but for this example it doesnt need one

File: src/components/nav-sidebar/NavSidebar.stories.js (hunk 2 above)
Comment (greg-in-a-box) [reply src/components/nav-sidebar/NavSidebar.stories.js:34]: no idea just a copy from the original

**Review by megansmith-box — APPROVED** [review]

**Review by tjuanitas — APPROVED** [review]

**Pull request merged**